### DISCUSSION TITLE
Add `espanso version` CLI command

### DISCUSSION THREAD

**Discussion by krystof-k** [espanso/espanso#1457] [Feature requests and Ideas]
It would be handy to quickly check the installed version.

**Comment by smeech** [comment]
Sorry about the late reply - I am trying to mop up and sort the old Issues and Discussions.

`espanso -V, --version    Prints version information`

**Reply by krystof-k** [reply]
Sure, no worries. I don't know how could I miss that.

**Reply by smeech** [reply]
No problem - just mopping up and closing discussions where I can.
